### PR TITLE
Address CVE-2022-41721

### PR DESCRIPTION
**Pull request by VeronikaSolovei9**
updated version of golang.org/x/net to v0.4.0

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -30,8 +30,8 @@ require (
 	github.com/vrischmann/go-metrics-influxdb v0.1.1
 	github.com/xeipuuv/gojsonschema v1.2.0
 	github.com/yudai/gojsondiff v1.0.0
-	golang.org/x/net v0.0.0-20220909164309-bea034e7d591
-	golang.org/x/text v0.3.7
+	golang.org/x/net v0.4.0
+	golang.org/x/text v0.5.0
```
Comment (mansinahar) [inline go.mod:34]: Thank you so much @VeronikaSolovei9 for putting out this fix PR! Any reason for pinning the text package version to `v0.5.0`? I see that the latest version for this package is `0.6.0`. Should we/can we move to the latest?

**Review by bsardo — COMMENTED** [review]
LGTM though I'm also curious about https://github.com/prebid/prebid-server/pull/2545/files#r1088081338.

File: go.mod (hunk 1 above)
Comment (SyntaxNode) [reply go.mod:34]: Yes, please move to the latest version available.

Comment (mansinahar) [reply go.mod:34]: Same for the /net package, the latest version is [v0.5.0](https://pkg.go.dev/golang.org/x/net?tab=versions). Can we please pin it to that?

**Comment by VeronikaSolovei9** [comment]
I put v0.4.0 because Trivy suggested to put this as a fixed version. I agree latest would be better. I checked it works with v0.5.0

**Review by mansinahar — APPROVED** [review]

**Pull request merged**